lib/hanami/providers/rack.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "dry/system/provider/source"
+
+module Hanami
+  module Providers
+    class Rack < Dry::System::Provider::Source
+      def prepare
+        require "dry/monitor/rack/middleware" # FIXME: this _used_ to be just "dry/monitor"
```

Comment:
This was changed to just `require "dry/monitor"` by @solnic in https://github.com/hanami/hanami/pull/1171, but then changed back by @jodosha in https://github.com/hanami/hanami/commit/c816219bcd38024c48b6b16bcd36a2cde365917f.

Now that dry-monitor has zeitwerk-enabled autoloading, I think we should leave it as `require "dry/monitor"`. @jodosha: was there any reason you changed it back?

Reply:
n.b. this does mean we'll need to make a dry-monitor release soon.

Reply:
@timriley Because with just `require "dry/monitor"` it was crashing. 🤷‍♂️